### ISSUE TITLE
Be able to force AP mode from command line

### ISSUE THREAD

**Issue by rohbotics** [rohbotics/pifi#9]
Maybe some kind of Unix Domain socket to tell the daemon to start AP mode?

**Comment by shoemakerlevy9** [comment]
Is the idea behind this that RPi's are not able to connect to WiFi and be in AP mode at the same time?
 And is that because the RPi only has 1 WiFi chip that can only be in one mode at a time? 

At least for development we use TP link routers so that the robots can be connected to WiFi and be broadcasting WiFi at the same time. The routers are cheap enough where they could be added to your BOM potentially. 
[(https://www.amazon.com/TP-Link-Wireless-Portable-Travel-Router/dp/B00TQEX8BO/ref=sr_1_3?ie=UTF8&qid=1542384630&sr=8-3&keywords=pocket+router)]

**Comment by rohbotics** [comment]
Yes, that is the idea here. Basically the wifi chip on the Pi does not support scanning/connecting while in AP mode.

I don't think we will be able to add the router to the BOM, but I agree that it is a useful tool for development. Maybe we should write up a guide on how to get it working on the robot?

**Comment by MoffKalast** [comment]
Solved by `sudo pifi rescan -y`?
